data/blog/remote-control-vscode-release-recap-sept-2023.mdx

```diff
@@ -0,0 +1,57 @@
+
+My colleague Alex made a video demonstrating how easy and powerful the updated extension is:
+
+<figure>
+  <iframe
+    width="560"
+    height="315"
+    src="https://www.youtube.com/embed/ASZm_HyrHIc?si=bStK4K3J-Qclj59E"
+    title="YouTube video player"
+    frameBorder="0"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+    allowfullscreen
+  ></iframe>
+</figure>
+
+### Funnel and Serve web flows to get you started
+
+**Parker**: It’s no secret that we here love [Funnel](https://tailscale.com/kb/1223/tailscale-funnel/) (the Tailscale tool that lets you share a port on your localhost with the whole internet) and [Serve](https://tailscale.com/kb/1242/tailscale-serve/) (the super-simple server that powers Funnel and also lets you share privately with just your tailnet). But not everyone has had a chance to get acquainted with these features, and the first time activating them may require some [adjustments to your settings or your ACLs](https://tailscale.com/kb/1223/tailscale-funnel/#tailnet-policy-file-requirement).
+
+For folks who try to use Serve or Funnel without all the right configuration in place, we’ve added [a web-based guide](https://tailscale.com/kb/1223/tailscale-funnel/#setup) to take you through these steps right when you need it. True story: I needed this guide the other day when trying to use Serve from a [tagged device](https://tailscale.com/kb/1068/acl-tags/) for the first time—and it solved my problem in seconds. So whether you’re a first-timer or literally a Tailscale employee, we’re hoping this new system will help you spend less time reading error messages and more time sharing with your tailnet or the whole internet.
+
+### Tailnet lock now in beta
+
+**Parker**: Tailscale is always thinking about your [security and privacy](https://tailscale.com/security/), and you can see it in the way we’ve designed our software. We use end-to-end WireGuard encryption for every tailnet connection, so we can never see your data. And we’ve given administrators a [device approval](https://tailscale.com/kb/1099/device-approval/) option, to review any new node that tries to log in to the network, which provides a level of security and convenience that works for most users.
+
+But for the most sensitive use cases, even that may not be enough. Tailscale is a hosted product, which gives us some technical ability, as the control plane, to inject or manipulate nodes in your tailnet. For users who are concerned about that threat angle, we provide tailnet lock—[now in beta](https://tailscale.com/blog/tailnet-lock-beta/)!—which requires that new node keys be verified by existing trusted nodes before other devices trust them. This will prevent any devices from being added to your tailnet unless and until they are vouces for by a signing key that you maintain and that Tailscale does not have access to.
```

Comment:
```suggestion
But for the most sensitive use cases, even that may not be enough. Tailscale is a hosted product, which gives us some technical ability, as the control plane, to inject or manipulate nodes in your tailnet. For users who are concerned about that threat angle, we provide tailnet lock—[now in beta](https://tailscale.com/blog/tailnet-lock-beta/)!—which requires that new node keys be verified by existing trusted nodes before other devices trust them. This will prevent any devices from being added to your tailnet unless and until they are vouched for by a signing key that you maintain and that Tailscale does not have access to.
```